parser_library/src/workspaces/workspace_configuration.cpp

```diff
@@ -615,6 +615,7 @@ void workspace_configuration::copy_diagnostics(const diagnosable& target) const
         pg.collect_diags();
         for (const auto& d : pg.diags())
             target.add_diagnostic(d);
+        pg.diags().clear();
```

Comment:
won't this cause the messages to disappear when the diagnostics are refreshed by an independent action?

Reply:
It seems they disappear when either proc_grps.json or pgm_conf.json is saved. I haven't noticed any other action with similar results so far.

Nonetheless, this seems like a behavior which is shipped with V1.4.0 and the original goal of this PR was just to go back to the same state.